CATALOG.md

```diff
@@ -516,7 +530,36 @@ Result Type|normative
 Suggested Remediation|Add a liveness probe to deployed containers
 Best Practice Reference|https://connect.redhat.com/sites/default/files/2022-05/Cloud%20Native%20Network%20Function%20Requirements%201-3.pdf Section 5.2.16, 12.1 and 12.5
 Exception Process|There is no documented exception process for this.
-Tags|common,telco
+Tags|common
+#### net-admin-capability-check
```

Comment:
I think you need to rebase then re-run `make build-catalog-md` to clear these up.